pkg/resource/certificate_authority_activation/hooks.go

```diff
@@ -112,13 +112,13 @@ func (rm *resourceManager) customFindCertificateAuthorityActivation(
 		if certificateAuthorityARN == *r.ko.Spec.CertificateAuthorityARN {
 			numFound++
 			if numFound > 1 {
-				status, found, err := unstructured.NestedString(item.Object, "status", "status")
+				status, found, err := unstructured.NestedString(item.Object, "spec", "status")
```

Comment:
How did this work previously?

Reply:
We weren't testing multiple CertificateAuthorityActivation resources previously.